public/components/trace_analytics/requests/queries/services_queries.ts

```diff
@@ -225,24 +208,9 @@ export const getServiceMetricsQuery = (
   map: ServiceObject,
   mode: TraceAnalyticsMode
 ) => {
-  const traceGroupFilter = new Set(
-    DSL?.query?.bool.must
-      .filter((must: any) => must.term?.['traceGroup'])
-      .map((must: any) => must.term.traceGroup) || []
+  const targetResource = [].concat(
```

Comment:
this logic no longer consider traceGroup filters in DSL anymore, does it change behavior of service metrics when filtering by trace group?

seems it would always consider all targetResources instead of the subset of targetResources that belong to services with traceGroups matching the user's traceGroup filter, which should cause a difference?

I'm ok if the difference it makes is not meaningful for users, just want to check if it's what we want